conformance/utils/suite/profiles.go

```diff
@@ -111,6 +111,16 @@ var (
 	}
 )
 
+// RegisterConformanceProfile allows downstream tests to register unique profiles that
+// define their own set of features
+func RegisterConformanceProfile(p ConformanceProfile) {
```

Comment:
I do not see any reference to this function; is it actually used anywhere?

Reply:
This is exported so it can be used downstream - see the PR message body you'll see the code snippet

```
suite.RegisterConformanceProfile(KnativeConformanceProfile)
```